src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderCursor.cs

```diff
@@ -487,7 +487,7 @@ private void ThreadProc()
                                     // REVIEW: Avoid allocating a string for every line. This would probably require
                                     // introducing a CharSpan type (similar to ReadOnlyMemory but based on char[] or StringBuilder)
                                     // and implementing all the necessary conversion functionality on it. See task 3871.
-                                    text = rdr.ReadLine();
+                                    text = rdr.ReadEntry();
```

Comment:
Recommend a flag for multi-line support.

Reply:
I feel it can share the same flag with EnableQuote, If we allow quote, then multi-line support is necessary to make sure TextLoader function well